### PR TITLE
feat(final review list survey and admin): add an extra edit meal button and the one checkbox option

### PR DESCRIPTION
**Pull request by mucc001**

### CODE REVIEW

File: apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue [hunk 1]
```diff
@@ -12,14 +25,36 @@
           :selected-food-in-meal="isSelectedFoodInMeal(meal.id)"
           @action="action"
         ></component>
-        <v-checkbox
-          v-if="review === 'checkbox'"
-          v-model="reviewed"
-          class="review-checkbox__checkbox pl-3"
-          label="Reviewed"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:19]: translate -> add string to prompts.json

File: apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue [hunk 2]
```diff
@@ -12,14 +25,36 @@
...
-          :value="meal.id"
-        ></v-checkbox>
+        <v-row class="pl-3">
+          <v-col v-if="review === 'checkbox'" cols="auto">
+            <v-checkbox
+              v-model="reviewed"
+              class="review-checkbox__checkbox"
+              label="Reviewed, nothing to add/change"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:33]: translate -> add string to prompts.json

File: apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue [hunk 3]
```diff
@@ -12,14 +25,36 @@
...
+              </v-btn>
+            </v-hover>
+          </v-col>
+        </v-row>
       </div>
+      <v-checkbox
+        v-if="review === 'onecheckbox'"
+        v-model="reviewed"
+        class="pl-3 review-checkbox__checkbox"
+        label="Reviewed, nothing to add/change"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:56]: translate -> add string to prompts.json

File: apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue [hunk 4]
```diff
@@ -13,14 +13,36 @@
               @action="action"
               @update:context-id="updateContextId"
             ></meal-item>
-            <v-checkbox
-              v-if="review === 'checkbox'"
-              v-model="reviewed"
-              class="review-checkbox__checkbox pl-3"
-              label="Reviewed"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:20]: translate -> add string to prompts.json

File: apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue [hunk 5]
```diff
@@ -13,14 +13,36 @@
...
-              :value="meal.id"
-            ></v-checkbox>
+            <v-row class="pl-3">
+              <v-col v-if="review === 'checkbox'" cols="auto">
+                <v-checkbox
+                  v-model="reviewed"
+                  class="review-checkbox__checkbox"
+                  label="Reviewed, nothing to add/change"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:21]: translate -> add string to prompts.json

File: apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue [hunk 6]
```diff
@@ -13,14 +13,36 @@
...
+                  </v-btn>
+                </v-hover>
+              </v-col>
+            </v-row>
           </div>
+          <v-checkbox
+            v-if="review === 'onecheckbox'"
+            v-model="reviewed"
+            class="pl-3 review-checkbox__checkbox"
+            label="Reviewed, nothing to add/change"
```
Comment (lukashroch) [inline apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:44]: translate -> add string to prompts.json

**Review by lukashroch — CHANGES_REQUESTED** [review]
@mucc001 can you please translate the strings?

They should go to prompt.js, to the `submit-prompt`, so the prompt is translatable without hardcoded text.

Also I don't see for admin to be able to select `onecheckbox`, you'll just need to add it to https://github.com/MRC-Epid-it24/intake24/blob/master/packages/common/src/prompts/prompts.ts#L105C1-L105C1

**Comment by mucc001** [comment]
> @mucc001 can you please translate the strings?
> 
> They should go to prompt.js, to the `submit-prompt`, so the prompt is translatable without hardcoded text.
> 
> Also I don't see for admin to be able to select `onecheckbox`, you'll just need to add it to https://github.com/MRC-Epid-it24/intake24/blob/master/packages/common/src/prompts/prompts.ts#L105C1-L105C1

Hi Lukas. Thank you for the comments. I have moved the label to recall.json. In addition, I added the relevant code to the admin.

**Pull request merged**